roles/provision-unifiedpush-apb/tasks/main.yml

```diff
@@ -131,6 +140,22 @@
           - "-i"
           - "-c"
           - 'wget http://localhost:8080/ag-push '
+        containers:
+    - name: ups-oauth-proxy
+      image: '{{ proxy_image }}:{{ proxy_image_tag }}'
+      imagePullPolicy: IfNotPresent
+      ports:
+      - containerPort: '{{ ups_proxy_port }}'
+        name: public
+      args:
+      - --provider=openshift
+      - --openshift-service-account={{ proxy_serviceaccount_name }}
+      - --upstream=http://localhost:{{ ups_port }}
+      - >-
+        --openshift-sar={"namespace":"{{ namespace }}","resource":"deploymentconfigs","name":"ups","verb":"update"}
```

Comment:
is this for the `update` permission ?

Reply:
So this is saying that the oauth proxy will only let users past if they have 'update' permission on the ups deploymentconfig.